docs/en/query_language/table_functions/index.md

```diff
@@ -1,8 +1,30 @@
 # Table functions
 
-Table functions can be specified in the FROM clause instead of the database and table names.
-Table functions can only be used if 'readonly' is not set.
-Table functions aren't related to other functions.
+Table function is a method of creating a table.
 
+You can use table functions in:
+
+* [CREATE TABLE](../create.md#create-table-query) query.
+
+    It's an alternative method of creating a table.
+
+* [FROM](../select.md#select-from) clause of the `SELECT` query.
+
+    The method of creating a temporary table, that is deleted after the query finishes.
```

Comment:
it's also unavailable from concurrent queries, so I'd just avoid mentioning "creating/deleting" in the first place